src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -676,48 +732,17 @@ export default class FilingHistoryList extends Mixins(
       case FilingTypes.INCORPORATION_APPLICATION:
       case FilingTypes.CHANGE_OF_REGISTRATION:
       case FilingTypes.REGISTRATION:
-        try {
-          // show spinner since the network calls below can take a few seconds
-          this.$root.$emit('showSpinner', true)
-
-          // fetch original filing
-          const origFiling = item.filingLink && await this.fetchFiling(item.filingLink)
-          if (!origFiling) {
-            throw new Error('Invalid API response')
-          }
-
-          // build correction filing
-          let correctionFiling: CorrectionFilingIF
-          if (this.isBComp) correctionFiling = this.buildIaCorrectionFiling(origFiling)
-          if (this.isFirm) correctionFiling = this.buildFmCorrectionFiling(origFiling)
-          if (!correctionFiling) throw new Error('Invalid filing type')
-
-          // save draft filing
-          const draftCorrection = await this.createFiling(this.getIdentifier, correctionFiling, true)
-          const draftCorrectionId = +draftCorrection?.header?.filingId
-          if (isNaN(draftCorrectionId)) {
-            throw new Error('Invalid API response')
-          }
-
-          // navigate to Edit web app to complete this correction
-          // NB: no need to clear spinner
-          const correctionUrl = `${this.editUrl}${this.getIdentifier}/correction/?correction-id=${draftCorrectionId}`
-          navigate(correctionUrl)
-        } catch (error) {
-          // clear spinner on error
-          this.$root.$emit('showSpinner', false)
-
-          // eslint-disable-next-line no-console
-          console.log('Error creating correction =', error)
-          this.loadCorrectionDialog = true
-        }
+        this.startCorrectionDialog(item)
         break
 
       case FilingTypes.CORRECTION:
         // FUTURE: allow a correction to a correction?
         // this.$router.push({ name: Routes.CORRECTION,
         //   params: { correctedFilingId } })
-        alert('At this time, you cannot correct a correction. Please contact Ops if needed.')
```

Comment:
Is there a requirement to do this?

As far as I am aware, we still do not support correcting a correction.

Reply:
You are right. I've changed it back.